### PR TITLE
syscalls: define the ABI

### PR DESCRIPTION
**Pull request by Stebalien**

### CODE REVIEW

**Comment by Stebalien** [comment]
I apologize if this makes absolutely no sense. Bit it's a start.

**Comment by raulk** [comment]
Closes https://github.com/filecoin-project/fvm-specs/issues/63.

**Review by raulk — COMMENTED** [review]
Oops, hadn't submitted my review. Mostly nits

File: 08-syscalls.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+- Signed integers are passed as two's-complement.
+
+## Conventions
+
+Conventionally:
+
+- Byte buffers & strings are passed as pointer/length pairs.
+- CIDs are passed as pointer only.
+- Token amounts are passed as two `u64` values representing the "high" and "low" bits of a `u128`
+  attoFIL value.
```
[the remark this answers is not in the document]
Comment (Stebalien) [reply 08-syscalls.md:42]: For parameters, it's high then low. Only "send" uses this, at the momoent.

**Comment by Stebalien** [comment]
We can iterate on this with future PRs, but I want to be able to link to it.

**Pull request merged**